src/Assets/TestProjects/KitchenSink/TestApp/TestApp.csproj

```diff
@@ -59,7 +59,7 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
       <CopyToPublishDirectory>Never</CopyToPublishDirectory>
     </Content>
-    
+
```

Comment:
We should be setting the `<PredefinedCulturesOnly>` property in the top PropertyGroup of this file.

Reply:
@dsplaisted I have returned back the changes in GivenThatWeWantToPublishAProjectWithAllFeatures.cs and TestApp as @eerhardt suggested to keep it to test the behavior when enabling the property in down-level version even if it will not have any runtime effect.